.github/workflows/_kotlin.yml

```diff
@@ -79,3 +79,33 @@ jobs:
         run: |
           echo -n "$FIREBASE_APP_DISTRIBUTION_CREDENTIALS" > $FIREBASE_CREDENTIALS_PATH
           ./gradlew --info appDistributionUploadRelease uploadCrashlyticsSymbolFileRelease
+
+  unit_test:
+    # Android SDK tools hardware accel is available only on Linux runners
+    runs-on: ubuntu-22.04
+    defaults:
+      run:
+        working-directory: ./kotlin/android
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./.github/actions/setup-rust
+        with:
+          targets: armv7-linux-androideabi aarch64-linux-android x86_64-linux-android i686-linux-android
+      - uses: actions/setup-java@v4
+        with:
+          distribution: temurin
+          java-version: 17
+      - uses: actions/checkout@v4
+      - uses: actions/setup-java@v4
+        with:
+          distribution: temurin
+          java-version: 17
+      - name: Validate Gradle wrapper
+        uses: gradle/wrapper-validation-action@v2
+      - name: Setup Gradle
+        uses: gradle/gradle-build-action@v3
+        with:
+          build-root-directory: ./kotlin/android
+      - run: touch local.properties
+      - name: Run tests
+        run: ./gradlew testReleaseUnitTest
```

Comment:
```suggestion
```

We already run this task: https://github.com/firezone/firezone/pull/4177/files#diff-7614a82db0df80ef0fe0ff5ecfde1cd3e47d020cbd29912c6554d269216290c3R66

Reply:
Woops! Nice catch!